asv/plugins/rattler.py

```diff
@@ -0,0 +1,141 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Do you need this separate notice?

Reply:
No, but it is present in almost every `.py` file in the repo (52/57 files to be exact), including a bunch of tests. Would be nice to get rid of it, but maybe project-wide instead of here?

Reply:
Actually still makes sense to remove it, and have a follow up for it later.